xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-ui/src/test/it/org/xwiki/test/ui/CompareVersionsTest.java

```diff
@@ -250,6 +251,8 @@ public void testAllChanges()
         assertDiff(ageDiff.getDiff("Pretty Name"), "+age");
         assertDiff(ageDiff.getDiff("Size"), "+30");
         assertDiff(ageDiff.getDiff("Number Type"), "+integer");
+
+        testVersionNavigation();
```

Comment:
No this not correct Enygma. I actually told Adel to do it this way since this is the way it should be. Functional tests need to have a single test method with a scenario. That's because they're slow to execute and you do not want to incur any setup penalty.

See http://dev.xwiki.org/xwiki/bin/view/Community/Testing#HBestpractices-2